1.1.0/haskell-language-server/tools/chocolateyinstall.ps1

```diff
@@ -45,13 +39,10 @@ function Install-Hls($ghcVersion) {
 }
 
 function Get-InstalledGhcVersions() {
-  $mbGhcs = choco list ghc --all --local --limit-output
+  $ghcExes = $(Get-Command ghc -All).path
```

Comment:
might consider adding `-CommandType Application` to avoid finding things that are a not executables.

Reply:
In this case it doesn't matter, but since you already took the dependency on the chocolatey extensions you could have used https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/extensions/chocolatey-core.extension/extensions/Get-AppInstallLocation.ps1